src/util/model_info.hpp

```diff
@@ -54,10 +54,15 @@ namespace big
 			return nullptr;
 		}
 
-		static bool is_model_of_type(const rage::joaat_t hash, const eModelType type)
+		template<typename T, typename ...Args>
+		static bool is_model_of_type(const rage::joaat_t hash, const T arg, const Args... args)
```

Comment:
Why is T a template argument? Why is `const T arg` never used?

Can you change `Args... args` to something more descriptive like `ModelTypes... types`?

Reply:
T exists to ensure at least one argument, since ...Args allows 0 arguments

Reply:
You are right, I forgot to use arg, just changed